app/components/Blockchain/Transaction/N3ContractInvocationAbstract.jsx

```diff
@@ -0,0 +1,37 @@
+// @flow
+import React from 'react'
+import classNames from 'classnames'
+import { injectIntl } from 'react-intl'
+import styles from './Transaction.scss'
+import InvokeIcon from '../../../assets/icons/raw-invoke-tx.svg'
+
+type Props = {
+  contract_name: string,
```

Comment:
Looks like eslint is failing here and below:

```
root/neon-wallet/app/components/Blockchain/Transaction/N3ContractInvocationAbstract.jsx
  17:13  error  Identifier 'contract_name' is not in camel case  camelcase
  30:12  error  Identifier 'contract_name' is not in camel case  camelcase
  ```

Reply:
This is a raw field name coming straight out of the `dora` api.  I've set the linter to ignore it instead of changing the name in the action for simplicity sake.